lwt/tls_lwt.ml

```diff
@@ -153,12 +153,12 @@ module Unix = struct
             handle tls (String.sub (Bytes.unsafe_to_string t.recv_buf) 0 n)
           | `Closed -> Lwt.return `Eof
 
-  let rec read t buf =
+  let rec read t ?(off = 0) buf =
```

Comment:
I guess we should add a check that `off` is non-negative and less than `Bytes.length buf`. Else we get an `Invalid_argument _` exception from `Bytes.blit_string` which is less clear.

Reply:
that's a great idea. I'll raise an exception..

Reply:
b9cb064